### PR TITLE
feat: added expo plugin

### PR DESCRIPTION
**Pull request by janrtvld**
Completes #13

### CODE REVIEW

**Review by morrieinmaas — CHANGES_REQUESTED** [review]
What about iOS?

File: app.plugin.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('./build/withAndroidPermissions.js')
```
[the remark this answers is not in the document]
Comment (janrtvld) [reply app.plugin.js:1]: It is exported from the build folder.

**Comment by janrtvld** [comment]
> What about iOS?

IOS doesn't require any additional config (according to the README).

File: app.plugin.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (morrieinmaas) [reply app.plugin.js:1]: ah yeah true

File: package.json [hunk 2]
```diff
@@ -41,6 +42,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "dependencies": {
+    "@expo/config-plugins": "^6.0.1",
```
Comment (blu3beri) [inline package.json:45]: Can this not be a dev dependency?

**Review by blu3beri — APPROVED** [review]
Looks good! As long as it's tested with an expo app I think we can merge it and release it.

**Review by morrieinmaas — APPROVED** [review]

**Pull request merged**